Mixpanel/MPNotificationViewController.m

```diff
@@ -151,33 +150,53 @@ - (void)buttonTapped:(UIButton *)button {
     [self.delegate notificationController:self wasDismissedWithCtaUrl:((MPTakeoverNotification *)self.notification).buttons[button.tag].ctaUrl];
 }
 
-- (void)hideWithAnimation:(BOOL)animated completion:(void (^)(void))completion
-{
-    [self.presentingViewController dismissViewControllerAnimated:animated completion:completion];
+- (void)show {
+    _window = [[UIWindow alloc] initWithFrame:CGRectMake(0, [UIScreen mainScreen].bounds.size.height, [UIScreen mainScreen].bounds.size.width, [UIScreen mainScreen].bounds.size.height)];
+    _window.windowLevel = UIWindowLevelAlert;
+    _window.rootViewController = self;
+    [_window setHidden:NO];
+
+    CGRect windowFrame = _window.frame;
+    windowFrame.origin.y -= [UIScreen mainScreen].bounds.size.height;
+
+    [UIView animateWithDuration:0.25 animations:^{
+        [self->_window setFrame:windowFrame];
```

Comment:
oh whoa. is there a reason we aren't doing this as a property? the `->ivar` notation is pretty non-standard at this point